### PR TITLE
chore(docs): update optimize update guides to use correct version

### PR DESCRIPTION
**Pull request by RomanJRW**
## Description

In line with the future Optimize versioning, this updates the update notes to use the future versioning system

## When should this change go live?

The version changes comes with the next minor release, but earlier docs changes are also fine to help users get used to the new scheme

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: optimize/self-managed/optimize-deployment/migration-update/instructions.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ title: "Instructions"
 description: "Find out how to update to a new version of Optimize without losing your reports and dashboards."
 ---
 
-Roughly every quarter of a year a new minor version of Optimize is released. These documents guide you through the process of migrating your Optimize from one Optimize minor version to the other.
+Optimize releases two new minor versions a year. These documents guide you through the process of migrating your Optimize from one Optimize minor version to the other.
```
Comment (akeller) [inline optimize/self-managed/optimize-deployment/migration-update/instructions.md:7]: This is probably a change we want to backport.

File: optimize/self-managed/optimize-deployment/migration-update/instructions.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ title: "Instructions"
...
-| 3.10        | You are on the latest version.                                    |
-| 3.0 - 3.9.x | Rolling update to 3.10                                            |
-| 2.0 - 2.7   | 1. Rolling update to 2.7 <br /> 2. Rolling update from 2.7 to 3.0 |
-| 1.0 - 1.5   | No update possible. Use the latest version directly.              |
+| Update from  | Recommended update path to 7.20/8.3                               |
+| ------------ | ----------------------------------------------------------------- |
+| 7.20/8.3     | You are on the latest version.                                    |
+| 3.0 - 3.10.x | Rolling update to 7.20/8.3                                        |
+| 2.0 - 2.7    | 1. Rolling update to 2.7 <br /> 2. Rolling update from 2.7 to 3.0 |
+| 1.0 - 1.5    | No update possible. Use the latest version directly.              |
```
Comment (akeller) [inline optimize/self-managed/optimize-deployment/migration-update/instructions.md:16]: Is this true for the update path to 7.20/8.3? Or are there other changes that need to be made in the migration instruction section to support this or the 3.7 to 7.20/8.3 path?

File: optimize/self-managed/optimize-deployment/migration-update/instructions.md (hunk 1 above)
Comment (misiekhardcore) [reply optimize/self-managed/optimize-deployment/migration-update/instructions.md:7]: Do you think we should backport this info to all versions?

**Comment by akeller** [comment]
@misiekhardcore let's update it there too. 3.11 won't exist as far as I'm aware.

File: optimize/self-managed/optimize-deployment/migration-update/instructions.md (hunk 1 above)
Comment (akeller) [reply optimize/self-managed/optimize-deployment/migration-update/instructions.md:7]: @misiekhardcore 😬 Do you know when Optimize changed the release cadence? 3.7+?

Comment (akeller) [reply optimize/self-managed/optimize-deployment/migration-update/instructions.md:7]: Add the backport in the latest commit.

**Review by misiekhardcore — APPROVED** [review]
From my side the changes look good right now

**Comment by akeller** [comment]
@misiekhardcore I'll merge this in. When @RomanJRW is back, we can review and iterate to make sure this looks good for the minor release 💪 thank you!

**Pull request merged**